examples/docs_beta_snippets/docs_beta_snippets/guides/components/index/26-duckdb-select-orders.txt

```diff
@@ -1,14 +1,12 @@
 duckdb /tmp/jaffle_platform.duckdb -c "SELECT * FROM orders LIMIT 5;"
 
-┌──────────┬─────────────┬────────────┬───┬───────────────┬──────────────────────┬──────────────────┬────────┐
-│ order_id │ customer_id │ order_date │ … │ coupon_amount │ bank_transfer_amount │ gift_card_amount │ amount │
-│  int32   │    int32    │    date    │   │    double     │        double        │      double      │ double │
-├──────────┼─────────────┼────────────┼───┼───────────────┼──────────────────────┼──────────────────┼────────┤
-│        1 │           1 │ 2018-01-01 │ … │           0.0 │                  0.0 │              0.0 │   10.0 │
-│        2 │           3 │ 2018-01-02 │ … │           0.0 │                  0.0 │              0.0 │   20.0 │
-│        3 │          94 │ 2018-01-04 │ … │           1.0 │                  0.0 │              0.0 │    1.0 │
-│        4 │          50 │ 2018-01-05 │ … │          25.0 │                  0.0 │              0.0 │   25.0 │
-│        5 │          64 │ 2018-01-05 │ … │           0.0 │                 17.0 │              0.0 │   17.0 │
-├──────────┴─────────────┴────────────┴───┴───────────────┴──────────────────────┴──────────────────┴────────┤
-│ 5 rows                                                                                 9 columns (7 shown) │
-└────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+┌──────────┬─────────────┬────────────┬───────────┬────────────────────┬───────────────┬──────────────────────┬──────────────────┬────────┐
+│ order_id │ customer_id │ order_date │  status   │ credit_card_amount │ coupon_amount │ bank_transfer_amount │ gift_card_amount │ amount │
+│  int32   │    int32    │    date    │  varchar  │       double       │    double     │        double        │      double      │ double │
+├──────────┼─────────────┼────────────┼───────────┼────────────────────┼───────────────┼──────────────────────┼──────────────────┼────────┤
+│        1 │           1 │ 2018-01-01 │ returned  │               10.0 │           0.0 │                  0.0 │              0.0 │   10.0 │
+│        2 │           3 │ 2018-01-02 │ completed │               20.0 │           0.0 │                  0.0 │              0.0 │   20.0 │
+│        3 │          94 │ 2018-01-04 │ completed │                0.0 │           1.0 │                  0.0 │              0.0 │    1.0 │
+│        4 │          50 │ 2018-01-05 │ completed │                0.0 │          25.0 │                  0.0 │              0.0 │   25.0 │
+│        5 │          64 │ 2018-01-05 │ completed │                0.0 │           0.0 │                 17.0 │              0.0 │   17.0 │
+└──────────┴─────────────┴────────────┴───────────┴────────────────────┴───────────────┴──────────────────────┴──────────────────┴────────┘
```

Comment:
this was auto-generated by the snapshot command, not sure what's happening here

Reply:
Yeah this happens to me too. Has to do with the size of the terminal in which you run the command-- we need to fix the width. See: https://dagsterlabs.slack.com/archives/C06U59R0N4A/p1739548697924479

cc @benpankow

Reply:
lol that's amazing